src/components/legacy/markdown/custom-components.tsx

```diff
@@ -117,6 +118,10 @@ const Figure = styled.figure`
   }
 `;
 
+const StyledCode = styled.code`
+  overflow-wrap: anywhere !important;
```

Comment:
For code formatting we use [PrismJS](https://www.gatsbyjs.com/plugins/gatsby-remark-prismjs/). For this there are some CSS files in the project. Maybe instead of the new `StyledCode` component and the `!important` you can add the CSS directly here. I have not tried it myself yet.

https://github.com/satellytes/satellytes.com/blob/9e091ad924196e9df2d92c0c4f32e743ead77ccb/src/assets/prism-themes/coy-with-shadows.scss#L33-L44

Reply:
Thank you for your suggestion, this is way cleaner, I was already wondering where the language css comes from.
Fixed it 👍